core/src/forwarding_stage.rs

```diff
@@ -292,11 +292,13 @@ impl<F: ForwardAddressGetter> ForwardingStage<F> {
             if packet.meta().is_simple_vote_tx() {
                 vote_batch.push((packet_data_vec, tpu_vote));
                 if vote_batch.len() == vote_batch.capacity() {
+                    self.metrics.votes_forwarded += vote_batch.len();
```

Comment:
https://github.com/anza-xyz/agave/pull/4829/files#diff-f87848d96da07f40ab18979056bf2b6a84f9b2f7e6c8af50dc7bbfb04ec52991L305

If you look a few lines below right after the
```// Send out remaining packets``` comment, is where we already accumulate.

Reply:
Yeah, makes total sense to me.